README.md

```diff
@@ -10,6 +10,8 @@ Details about Nemo and its development can be found in:
 * Our website: https://nemo.apache.org/
 * Our project wiki: https://cwiki.apache.org/confluence/display/NEMO/
 
```

Comment:
Can we add our mailing list information over here? I understand that it is not entirely pertinent to this PR, but the change is minor :)

Reply:
Sure, I'll also add a link for subscription.